nmstate.gpg

```diff
@@ -0,0 +1,102 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
Since it is ascii-armored, it should probably be `nmstate.asc` instead of `nmstate.gpg`

Reply:
The `asc` is only indicate its a ASCII file, while `gpg` indicate it is related to GPG.

The Fedora is also using `fedora.gpg` for this:

https://getfedora.org/en/security/

Reply:
ok, fine with me